### DISCUSSION TITLE
How to update the hostname of servers?

### DISCUSSION THREAD

**Discussion by mike-boost** [gravitational/teleport#18632] [Q&A] [answered]
When running **tctl nodes ls**:
Only the **Labels** will update when the server's hostname is changed. The **Host** name is outdated.

**Comment by webvictim** [comment, marked as answer]
I think the hostname is only read on startup. You'd likely need to restart or reload the Teleport process on the server to detect a change.

**Comment by NikhilTM3** [comment]
To change the hostname we need to change it in the node.
•	sudo hostname “abc”
•	sudo vi /etc/hostname ---> Change the hostname in this file
•	vi /etc/teleport.yaml
Now reload teleport
•	systemctl reload teleport
